src/private-route/PrivateRoute.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from "react";
+import { Route } from "react-router-dom";
+
+import { useAuth0 } from "../react-auth0-spa";
+
+const PrivateRoute = ({ component: Component, path, ...rest }) => {
+  const { loading, isAuthenticated, loginWithRedirect } = useAuth0();
+
+  useEffect(() => {
+    if (loading || isAuthenticated) {
+      return;
+    }
+    const fn = async () => {
+      await loginWithRedirect({
+        appState: { targetUrl: window.location.pathname }
+      });
+    };
+    fn();
+  }, [loading, isAuthenticated, loginWithRedirect, path]);
```

Comment:
Would prefer this instead to be a react hook that private pages can use. Then, the routing system doesn't need to treat private pages specially -- keeping making separate concerns separate.

Also, this file can go into the same folder as whatever folder `react-auth0-spa.js` ends up being in!

Reply:
I filed https://github.com/Recidiviz/covid19-dashboard/issues/27 to do this in the future. Good suggestion.